apioak/admin/project.lua

```diff
@@ -2,6 +2,7 @@ local db         = require("apioak.db")
 local pdk        = require("apioak.pdk")
 local schema     = require("apioak.schema")
 local controller = require("apioak.admin.controller")
+local _
```

Comment:
Why is that?

Reply:
Because '_' is used many times in the code to make a unified definition。

Reply:
`_` this is an ignored variable placeholder, normally it does not require explicit declaration.

http://lua-users.org/wiki/LuaStyleGuide

https://github.com/mpeterv/luacheck/issues/40

Reply:
OK, thanks.

Reply:
LGTM, cool